package.json

```diff
@@ -11,24 +11,36 @@
     "bootstrap-datepicker": "1.4.1",
     "c3": "0.4.17",
     "d3": "3.5.17",
+    "fbjs": "0.8.14",
+    "history": "4.7.2",
+    "hoist-non-react-statics": "2.3.1",
+    "invariant": "2.2.2",
```

Comment:
The number of real node micro-dependencies used during development signals a departure from the prior javascript dependencies of Cockpit ... which mostly brought in browser-ready javascript libraries. 

This isn't a blocker, but likely needs to be discussed with the team more broadly before this is merged.

Reply:
In fact, only `react-router` and `react-router-dom` are directly addresses within this PR.

The remaining ones were added to pass semaphore CI. It was complaining about omitting to pull peer-dependencies.
Locally executed `./autogen.sh && ./tools/make-rpms` passes well even without adding those micro-dependencies.

Reply:
@mareklibra let's see if we can fix semaphore. I can give it a shot, do you have the errors it gave handy?